halo/cmd/ready.go

```diff
@@ -16,7 +16,7 @@ type readyConfig struct {
 
 func defaultReadyConfig() readyConfig {
 	return readyConfig{
-		MonitoringURL: "http://localhost:26660",
+		MonitoringURL: "http://localhost:26660/ready",
```

Comment:
I'd argue that `MonitoringURL` should only be the base host and port of the monitoring API to query. The `ready` command should add the path it wants to query to in that base URL

Reply:
Agreed, I had the same first intuition, but then remembered the "remote node" discussion and was not sure anymore. Will make it a non-configurable part of the URL.